### PR TITLE
change env naming convention

### PR DESCRIPTION
**Pull request by momentmaker**

### CODE REVIEW

**Review by tateexon — COMMENTED** [review]
I think you will need to update the constants here https://github.com/smartcontractkit/chainlink-terra/blob/ddd0db809d7e89bedd7edf8bb196b72ed28dae77/packages-ts/gauntlet-terra-contracts/src/lib/constants.ts or you will break some things like the gauntlet `token:deploy` command.

**Comment by momentmaker** [comment]
@tateexon awesome! thank you for the heads up. I wasn't aware of those. could you please review again?

File: packages-ts/gauntlet-terra-contracts/codeIds/bombay-testnet.json [hunk 1]
```diff
@@ -1,9 +0,0 @@
-{
```
Comment (tateexon) [inline packages-ts/gauntlet-terra-contracts/codeIds/bombay-testnet.json:1]: This just needed renamed, not deleted right?

**Review by tateexon — CHANGES_REQUESTED** [review]

File: packages-ts/gauntlet-terra-contracts/codeIds/bombay-testnet.json (hunk 1 above)
Comment (momentmaker) [reply packages-ts/gauntlet-terra-contracts/codeIds/bombay-testnet.json]: oops. I had renamed it but git add didn't automatically pick it up

**Review by tateexon — APPROVED** [review]
Looks good as far as I can tell. I at least don't know of anything else it could break.

**Pull request merged**